File: doordeck-sdk/src/mingwMain/resources/csharp/Model/Context.cs
namespace Doordeck.Headless.Sdk.Model;

public class OperationContextData(
    string userId,
    string userCertificateChain,
    string userPublicKey,
    string userPrivateKey)
{
    public string UserId { get; set; } = userId;
    public string UserCertificateChain { get; set; } = userCertificateChain;
    public string UserPublicKey { get; set; } = userPublicKey;
    public string UserPrivateKey { get; set; } = userPrivateKey;
}

File: doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/model/Context.kt
package com.doordeck.multiplatform.sdk.model

import kotlinx.serialization.Serializable

object Context {

    @Serializable
    data class OperationContextData(
        val userId: String,
        val userCertificateChain: String,
        val userPublicKey: String,
        val userPrivateKey: String
    )
}